docs/user/gcp/install_upi.md

```diff
@@ -24,6 +24,12 @@ Use a [staged install](../overview.md#multiple-invocations) to enable desired cu
 ### Create an install config
 Create an install configuration as for [the usual approach](install.md#create-configuration).
 
+If you are installing into a Shared VPC, the installer will not be able to find the base domain zone in the host project.
+You must either create the install-config another way, or select a zone in the service project and then update the `install-config.yaml` manually.
+However, both methods may cause the base domain verification to fail later when you `create manifests` if you are not selecting to `publish: Internal`.
+Alternatively, you can make a dummy DNS zone in the service project that has the same name and domain as the one in the host project.
+This dummy zone will not be needed after the Ignition files have been created.
```

Comment:
a) make sure we make it clear that Shared VPC (XPN) is different from Sharing VPC in the same project

Reply:
b) This dummy workarounds shouldn't be what we document for our users, if I saw something like that in _some project_ I would rather not use that project. Clearly state the restriction that only Internal will work for XPN.

Reply:
Updated with 998a518a17e54a792bd52cf31b217775eb20f5a3